tests/handlers/detector_data_handler_test.py

```diff
@@ -95,6 +95,25 @@ def test_factory_returns_none_for_unknown_stream_kinds() -> None:
     assert preprocessor is None
 
 
+def test_factory_returns_none_for_unconfigured_detectors() -> None:
```

Comment:
This is testing the case where "Skip detectors that are not configured", but it seems we are missing a test for the "Skip log data for sources not in the attribute registry" case which was also fixed in this PR?